docs/getting-started/libnetwork/EnvironmentSetup.md

```diff
@@ -106,12 +106,12 @@ directory.
 
 ### Preload the Calico docker image (optional)
 
-You can optionally preload the Calico Docker image to avoid the delay when you 
-run `calicoctl node` the first time.  Select the same version of the Calico 
-Docker image as you selected above.  For example, to pull the latest released 
-version
+The libnetwork release of calico uses its own calico/node-libnetwork Docker 
```

Comment:
Strictly Calico (with a cap C) is really the networking component (i.e. Felix), so there isn't a libnetwork release of Calico.  There is of calico/node ... 

Anyway, I don't think you need to compare the two different images here - it just adds confusion.